src/ServiceStack.Text/PclExport.NetStandard.cs

```diff
@@ -83,6 +84,15 @@ public NetStandardPclExport()
 #else 
             this.DirSep = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? '\\' : '/';
 #endif
+            var req = HttpWebRequest.Create("http://servicestack.net");
+            try
+            {
+                req.Headers[HttpRequestHeader.UserAgent] = "ServiceStack";
```

Comment:
No it won't prevent from removing. It checks the exception on setting the restricted header and if it's raised that means that the property was removed and we should not try to set this header. The only way to prevent linker from removing the property (if do not change linker settings) it is directly reference the property (such as `req.UserAgent = "ServiceStack"`), but this can be done only in non - NetStandard projects.